packages/office-addin-manifest/src/manifestOperations.ts

```diff
@@ -49,15 +48,15 @@ export namespace OfficeAddinManifest {
 async function getManifestHandler(manifestPath: string): Promise<ManifestHandler> {
   let manifestHandler: ManifestHandler;
   if (manifestPath.endsWith(".json")) {
-    manifestHandler = new ManifestHandlerJson();
+    manifestHandler = new ManifestHandlerJson(manifestPath);
   } else if (manifestPath.endsWith(".xml")) {
-    manifestHandler = new ManifestHandlerXml();
+    manifestHandler = new ManifestHandlerXml(manifestPath);
   } else {
     const extension: string = manifestPath.split(".").pop() ?? "<no extension>";
     throw new Error(
       `Manifest operations are not supported in .${extension}.\nThey are only supported in .xml and in .json.`
     );
   }
-  await manifestHandler.readFromManifestFile(manifestPath);
```

Comment:
is readFromManifestFile being used anywhere anymore?

Reply:
This was just a side effect of the name change. The function is still being used here

Reply:
Sorry . . . I was mixing this up with "readManifestFile" which is in manifestOperations instead the handler's "readFromManifestFile".  It looked like it was just used by the tests now . . . but I just checked and it's exported outside the package (not just internally).  So we're good.